### ISSUE TITLE
Critical JS error when using Multiple Links inside Vizy

### ISSUE THREAD

**Issue by tomkiss** [verbb/hyper#60]
### Describe the bug

When enabling "multiple links" in a hyper field, inside Vizy, a Javascript error occurs and the field fails to load.

Error:

```
hyper-f6b61f82.js:18 TypeError: this.proxyValue.forEach is not a function
    at Proxy.created (HyperInput.vue:145:25)
    at Vt (hyper-f6b61f82.js:18:15354)
    at nt (hyper-f6b61f82.js:18:15424)
    at Ya (hyper-f6b61f82.js:18:39095)
    at yb (hyper-f6b61f82.js:18:38376)
    at Qu (hyper-f6b61f82.js:18:65673)
    at Hb (hyper-f6b61f82.js:18:65135)
    at Ju (hyper-f6b61f82.js:18:64774)
    at F (hyper-f6b61f82.js:18:52359)
    at D (hyper-f6b61f82.js:18:52251)
Hv @ hyper-f6b61f82.js:18
```
<img width="894" alt="Screenshot 2023-05-11 at 12 20 13" src="https://github.com/verbb/hyper/assets/183310/9638d207-e099-4219-8adf-3fcece53d60a">

### Steps to reproduce

1. Install Vizy and Hyper.
2. Create a Hyper field with default settings but enable "Multiple Links".
3. Create a Vizy field with default settings. Then choose Add Group > "Hyper", Add Block > "Hyper", and insert the Hyper field you just created.
4. Observe the Vizy field in an edit entry page and check browser console.

### Craft CMS version

4.4.10.1

### Plugin version

1.1.5 and Vizy 2.1.4 (also tested Vizy dev-craft-4)

### Multi-site?

No

**Comment by engram-design** [comment]
Should be fixed for the next release. To get this early run `composer require verbb/hyper:"dev-craft-4 as 1.1.5"`

**Comment by tomkiss** [comment]
Yep, that fixes it - thanks! 🙏

**Comment by bleepsandblops** [comment]
I'm getting the same error message on 1.1.5 even without multiple links, and if I use `dev-craft-4 as 1.1.5` I get
![Screenshot 2023-05-11 at 17 57 04](https://github.com/verbb/hyper/assets/1942313/75ccbeab-bc71-41bd-a0eb-3bd3930a99a5)

**Comment by engram-design** [comment]
@bleepsandblops have you tried clearing your caches (Utilities > Caches). That fact that it’s passing in [Object] as a string still makes me think that the Twig field template hasn’t been updated with the [changes](https://github.com/verbb/hyper/commit/94ea7af29bd3311c764f4892e44b1b417aeb0684#diff-b6829cf83d76a24f89481b4834ac4c4327bd53f330892613482ba5c37c02834a)

**Comment by bleepsandblops** [comment]
@engram-design I stand corrected! That fixed it, thank you.

**Comment by engram-design** [comment]
Fixed in [1.1.6](https://github.com/verbb/hyper/releases/tag/1.1.6)
